crypto/des/asm/crypt586.pl

```diff
@@ -6,9 +6,6 @@
 # in the file LICENSE in the source distribution or at
 # https://www.openssl.org/source/license.html
 
-# The inner loop instruction sequence and the IP/FP modifications are from
-# Svend Olaf Mikkelsen <svolaf@inet.uni-c.dk>
```

Comment:
I'd only omit e-mail address... [Applies to multiple cases.]

Reply:
Added the comment (without email) back, in two places.